packages/flatpickr/src/input.css

```diff
@@ -0,0 +1,898 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
```

Comment:
I think we can remove those here?

Reply:
Or even: It definitely shouldn't not be included as this has lots of side effects.

Reply:
I had to add these to be able able to use tailwind in the file. I don't think it would work without it. Let's have a call to go through this maybe?